api_v3/lib/types/EntryServerNode/live/KalturaLiveEntryServerNode.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * @package api
+ * @subpackage objects
+ */
+class KalturaLiveEntryServerNode extends KalturaEntryServerNode {
```

Comment:
This is since it can have multiple of each one and we don't want multiple DB lines for the same node